lib/modules/accountSwitcher.js

```diff
@@ -28,12 +23,12 @@ module.options = {
 	},
 	accounts: {
 		type: 'table',
-		addRowText: '+add account',
+		addRowText: i18n('accountSwitcherAddAccount'),
```

Comment:
Ideally we don't want to make i18n calls within the actual modules themselves, but rather in the backend where they are rendered. This might need some changes to module builder @andytuba @erikdesjardins 

Same for all below.